main.go

```diff
@@ -79,7 +79,7 @@ func init() {
 	flag.StringVar(&dbNameOverride, "dbname", "", "dbname: name to use for Spanner DB")
 	flag.StringVar(&instanceOverride, "instance", "", "instance: Spanner instance to use")
 	flag.StringVar(&filePrefix, "prefix", "", "prefix: file prefix for generated files")
-	flag.StringVar(&driverName, "driver", "", "driver name: experimental flag for accessing source DB via database/sql driver (accepted values are \"postgres\" and \"mysql\")")
+	flag.StringVar(&driverName, "driver", "", "driver name: experimental flag for accessing source DB or dump files (accepted values are \"pgdump\", \"postgres\", \"mysqldump\", and \"mysql\")")
```

Comment:
Drop experimental.

Also, suggest line 74 be:

	driverName       = "pgdump"

git is being difficult and won't allow me to put a comment there (weird -- maybe too far from the changes in file?)